lib/plugins/helper/list_tags.js

```diff
@@ -25,13 +25,19 @@ function listTagsHelper(tags, options) {
     ulClass = options.class.ul || `${className}${classStyle}`;
     liClass = options.class.li || `${className}${classStyle}-item`;
     aClass = options.class.a || `${className}${classStyle}-link`;
+    labelClass = options.class.label || `${className}${classStyle}-label`;
     countClass = options.class.count || `${className}${classStyle}-count`;
+
+    labelSpan = !!Object.prototype.hasOwnProperty.call(options.class, 'label');
```

Comment:
```suggestion
    labelSpan = Object.prototype.hasOwnProperty.call(options.class, 'label');
```

IMHO `!!` is not necessary.

Reply:
I course there are not and I never put them, that's again a post-commit hook. Those are dangerous and it's not the first time of of those hook mess with the code.